codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/DirectedRubyCodegen.java

```diff
@@ -75,7 +75,8 @@ public GenerationContext createContext(CreateContextDirective<RubySettings, Ruby
         Model model = directive.model();
         List<RubyIntegration> integrations = directive.integrations().stream()
                 .filter((integration) -> integration
-                        .includeFor(service, model))
+                        .includeFor(service, model)
```

Comment:
I think it should be one or the other. If we still need the other approach then maybe we shouldn't do this one.